hs_discover/public/index.html

```diff
@@ -1,17 +1,218 @@
-{% extends "base.html" %}
-{% load pages_tags mezzanine_tags keyword_tags comment_tags hydroshare_tags %}
-{% block meta_title %}Sample{% endblock %}
+{% extends "newbase.html" %}
```

Comment:
extensive html template tech debt. This is the start of cleanup/optimization and backing out of the approach of "one collossal mess of a global scope and directives"

Reply:
Could you speak to the work done in this approach?  I'm not able to understand the improvements easily.  I ran a diff and was surprised to find out newbase.html is nearly 200 lines larger than base.html.  If you're trying to fix the "one collossal mess of a global scope and directives" I'd expect that global scope to be trimmed, not seemingly exploded.

Reply:
Combed the includes so the file can be searched and deduplication can begin. Previously, had to have several files open and search across them and deduplication was impossible.

Reply:
"This is the start of cleanup/optimization" if you had a mess of dependencies spread across five rooms and had to continually move between rooms and remember what was where... you'd first want to get a warehouse, garage or airplane hangar and bring all the pieces to that area and start assembling, categorizing, deduplicating, optimizing, updating libraries, and pulling into properly scoped sections, while updating to modern standards, such as eslint like I've done many other places. Also, trimming down on Directives has begun and is still underway.